swagger.yaml

```diff
@@ -223,28 +253,114 @@ definitions:
       gap:
         description: "Mandatory gaps in trail (must be cleared)"
         type: "boolean"
-      
+
   Trail:
     type: "object"
     required:
     - "name"
     - "location"
     properties:
-      id:
-        type: "integer"
-        format: "int64"
-      name:
-        type: "string"
-        example: "Dan's Trail"
-      difficulty:
-        type: "string"
-        enum:
-        - "Green"
-        - "Blue"
-        - "Black"
-        - "Double Black"
-      features:
-        $ref: "#/definitions/Features"
-      location:
-        description: "Zip code of trail head"
-        type: "integer"
\ No newline at end of file
+      data:
+        properties:
+          id:
+            type: "integer"
+            format: "int64"
+          name:
+            type: "string"
+            example: "Dan's Trail"
+          zipCode:
+            description: "Zip code of trail head"
+            type: "integer"
+          difficulty:
+            type: "string"
+            enum:
+            - "Green"
+            - "Blue"
+            - "Black"
+            - "Double Black"
+          largeDrop:
+            description: "Drops larger than 4 feet"
+            type: "boolean"
+          smallDrop:
+            description: "Drops smaller than 4 feet"
+            type: "boolean"
+          woodRide:
+            description: "Any built wooden feature"
+            type: "boolean"
+          skinny:
+            description: "Any narrow ride at a height"
+            type: "boolean"
+          largeJump:
+            description: "Jumps taller than 4 feet"
+            type: "boolean"
+          smallJump:
+            description: "Jumps shorter than 4 feet"
+            type: "boolean"
+          gap:
+            description: "Mandatory gaps in trail (must be cleared)"
+            type: "boolean"
+
+  Trails:
+    type: "object"
+    properties:
+      data:
+        type: "array"
+        items:
+          type: "object"
+          properties:
+            id:
+              type: "integer"
+              format: "int64"
+            name:
```

Comment:
These attributes are copied between the trail and trails objects. Could you create a single object for both of these objects to keep it more DRY?